arch/arc/Kconfig

```diff
@@ -104,6 +105,18 @@ config FP_FPU_DA
 
 menu "ARC CPU Options"
 
+config ARC_LACKS_ZOL
```

Comment:
Wonder if it would be better as `ARC_HAS_ZOL`. It should be conceptually more inline with others like `ARCH_HAS_*`.

Reply:
Yep, make sense. Fixed for now, thanks.